tests/suites/test_suite_ecp.data

```diff
@@ -426,10 +426,26 @@ ECP test vectors Curve25519
 depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
 ecp_test_vec_x:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"057E23EA9F1CBE8A27168F6E696A791DE61DD3AF7ACD4EEACC6E7BA514FDA863":"47DC3D214174820E1154B49BC6CDB2ABD45EE95817055D255AA35831B70D3260":"6EB89DA91989AE37C7EAC7618D9E5C4951DBA1D73C285AE1CD26A855020EEF04":"61450CD98E36016B58776A897A9F0AEF738B99F09468B8D6B8511184D53494AB"
 
-ECP point multiplication Curve25519
+ECP point multiplication Curve25519 (normalized) #1
 depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
 ecp_test_mul:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"9":"0":"1":"057E23EA9F1CBE8A27168F6E696A791DE61DD3AF7ACD4EEACC6E7BA514FDA863":"0":"1":0
 
+ECP point multiplication Curve25519 (not normalized) #2
+depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
+ecp_test_mul:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"27":"0":"3":"057E23EA9F1CBE8A27168F6E696A791DE61DD3AF7ACD4EEACC6E7BA514FDA863":"0":"1":MBEDTLS_ERR_ECP_INVALID_KEY
+
+ECP point multiplication Curve25519 (element of order 2: origin) #3
+depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
+ecp_test_mul:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"0":"0":"1":"0":"0":"1":MBEDTLS_ERR_MPI_NOT_ACCEPTABLE
+
+ECP point multiplication Curve25519 (element of order 4: 1) #4
+depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
+ecp_test_mul:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"1":"0":"1":"0":"0":"1":MBEDTLS_ERR_MPI_NOT_ACCEPTABLE
+
+ECP point multiplication Curve25519 (element of order 8) #5
+depends_on:MBEDTLS_ECP_DP_CURVE25519_ENABLED
+ecp_test_mul:MBEDTLS_ECP_DP_CURVE25519:"5AC99F33632E5A768DE7E81BF854C27C46E3FBF2ABBACD29EC4AFF517369C660":"B8495F16056286FDB1329CEB8D09DA6AC49FF1FAE35616AEB8413B7C7AEBE0":"0":"1":"0":"0":"1":MBEDTLS_ERR_MPI_NOT_ACCEPTABLE
```

Comment:
Please add where the test vectors come from in the commit message. Also, consider using the correct answer, even though it technically doesn't matter as `nP` won't be used, for negative tests.